src/components/NativeDefinition/NativeDefinition.tsx

```diff
@@ -16,8 +18,13 @@ export interface NativeDefinitionProps extends Omit<TypographyProps, 'children'>
 }
 
 function NativeDefinition({ name, params, returnType, sx, noWrap = false, nameCopyable = true, ...rest }: NativeDefinitionProps) {
-  const { nativeDisplayMode, displayVoidReturnType } = useSettings()
-  const nameWithBreaks = useMemo(() => name.replace(/_/g, '_\u200b'), [name])
+  const { nativeDisplayMode, nativeTypes, displayVoidReturnType } = useSettings()
+  const nameWithBreaks = useMemo(() => {
+    if (nativeDisplayMode === 'TS') {
+      return toPascalCase(name)
```

Comment:
Adding a `\u200b` after each capital letter might result in cleaner line breaks when the name is getting text wrapped.